app/upgrade/fork.go

```diff
@@ -0,0 +1,60 @@
+package upgrade
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	stakingkeeper "github.com/cosmos/cosmos-sdk/x/staking/keeper"
+	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+
+	v046 "github.com/Pylons-tech/pylons/x/pylons/migrations/v046"
+)
+
+// CreateUpgradeHandler make upgrade handler
+func CreateUpgradeHandler(mm *module.Manager, configurator module.Configurator, staking *stakingkeeper.Keeper, pylonStoreKey storetypes.StoreKey, cdc codec.Codec) upgradetypes.UpgradeHandler {
+	return func(ctx sdk.Context, plan upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
+		newVM, err := mm.RunMigrations(ctx, configurator, vm)
+		if err != nil {
+			return newVM, err
+		}
+		FixMinCommisionRate(ctx, staking)
```

Comment:
"CommissionRate" would be a better spelling for the function.

Reply:
thanks @mustafapylons